### DISCUSSION TITLE
Provide a light project skeleton, like Symfony offers for minimal API projects

### DISCUSSION THREAD

**Discussion by madalinignisca** [laravel/framework#42373] [Ideas]
I love the way Laravel kept for getting started, especially for beginners.

But, for some of us, mainly working on SaaS applications, the laravel/laravel skeleton is not really friendly to quickly get started. Even to make a demo to somebody how easy is to get started with some common requirements of SaaS apps, involves a few minutes of changing the defaults.

On the other side, Symfony offers 2 project skeletons.
One is for minimal to get started, which is wonderful to quickly start an API and bring in the way that the customer needs to get Authentication done, and in my case, all fall in getting really quickly with OAUTH2.
The other one is a full web version, which builds on top of the first, so they don't spend time double maintain both.

It would be wonderful to see an official minimal skeleton for Laravel that doesn't enforces Sanctum and Sails in it's dependencies to get started and to which if u need passport, you avoid the time to cleanup.

Myself, I've been using a personal fork for doing quick demos in presentations: https://github.com/madalinignisca/laravel/tree/9.x-light, but I thought if this would be officially provided, there would be a lot more people enjoying it.

**Comment by madalinignisca** [comment]
PS. In my case, Sanctum is out of the question, as I start always with Passport. My SaaS applications allow external approved providers to integrate their apps for the users that want to use them, and Passport has always been a joy and bless for my team.

**Comment by shaedrich** [comment]
There is some work on a light project skeleton in general: #47309
